src/js/components/Calendar/__tests__/__snapshots__/Calendar-test.js.snap

```diff
@@ -8,8 +8,8 @@ exports[`Calendar date 1`] = `
   flex: 0 0 auto;
   width: 24px;
   height: 24px;
-  fill: #666666;
-  stroke: #666666;
+  fill: #444444;
```

Comment:
As I commented on the previous PR, the Calendar component has breaking changes due to the snapshot changes. the fix/PR should accommodate backward compatibility.

Reply:
Yes same for carousel need to figure out why it is changing these snap shots

Reply:
@ShimiSun I updated snap shots with master but still need to dig into calendar and carousel but now my snap shots are up to date!